README.md

```diff
@@ -8,8 +8,7 @@ repo](https://github.com/flutter/flutter). It contains the source code for
 Flutter first-party plugins (i.e., plugins developed by the core Flutter team).
 Check the `packages` directory for all plugins.
 
-Flutter plugins enable access to platform-specific APIs. For more information
-about plugins, and how to use them, see
+Flutter plugins enable access to platform-specific APIs. For more information about plugins and how to use them, see
```

Comment:
This is a purely subjective stylistic change.

Reply:
I have removed an unnecessary comma here. If you wish, I could revert it.

Reply:
I understand the change you made, I'm saying it's a purely subjective stylistic change. There was nothing wrong with it before, it just changes the emphasis very slightly. Unlike the typo fix in the other file, this isn't fixing a problem, it's just making it match how you would choose to write it.

Please don't make purely stylistic changes to docs. If everyone did that, we'd be constantly churning docs for no reason.